src/main/java/de/tum/in/www1/artemis/service/export/DataExportExerciseCreationService.java

```diff
@@ -316,7 +316,11 @@ private void createResultsAndComplaintFiles(Submission submission, Path outputDi
                     if (feedback != null) {
                         resultScoreAndFeedbacks.append("- Feedback: ");
 
-                        // null if it's manual feedback
+                        // null if it's manual feedback or if the test name was filtered out above
+                        if (feedback.getTestCase() != null && feedback.getTestCase().getTestName() != null) {
+                            resultScoreAndFeedbacks.append(feedback.getTestCase().getTestName()).append("\t");
+                        }
+                        // e.g the file and line number for programming exercises
```

Comment:
This comment describes an example of `feedback.getText()` content, right? That was not immediately clear to me. Can you add this information? Something like:
```suggestion
                        // feedback.getText() describes e.g the file and line number for programming exercises
```